<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Turma</class>
 <widget class="QWidget" name="Turma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableTurma">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>601</width>
     <height>331</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>idTurma</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>nomeTurma</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>turno</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>idProfessor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>idDisciplina</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="form">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Manipulação</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>461</width>
      <height>69</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelnomeTurma">
         <property name="text">
          <string>Nome da Turma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="linenomeTurma"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelTurno">
         <property name="text">
          <string>Turno</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTurno">
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelidProfessor">
         <property name="text">
          <string>Id do Professor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineProfessor"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0">
       <item>
        <widget class="QLabel" name="labelidDisciplina">
         <property name="text">
          <string>Id da Disciplina</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineDisciplina">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="labelData">
         <property name="text">
          <string>Data da Turma</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineData"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>151</width>
      <height>92</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnInserir">
       <property name="text">
        <string>Inserir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExcluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelBusca">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineBusca"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
